frontend/app/src/components/helper/DataTable.vue

```diff
@@ -148,5 +214,28 @@ const scrollToTop = () => {
   }
 }
 
+.items-page-select {
+  display: flex;
+
+  :deep() {
```

Comment:
Btw I think we are using the new `:deep()` selector wrong.

https://vuejs.org/api/sfc-css-features.html#deep-selectors

I am getting some warnings in WebStorm but also the example suggests `:deep(.v-input) {...}`